app/blueprints/assessments/templates/assessor_tasklist.html

```diff
@@ -147,41 +150,69 @@
             {% endif %}
         {% endfor %}
 
-        {% if sub_criteria_status_completed and g.access_controller.has_any_assessor_role and (not is_qa_complete) and (not flag_status) and not is_expression_of_interest %}
-            {{ assessment_complete(state, form.csrf_token, application_id)}}
+    {% if sub_criteria_status_completed and g.access_controller.has_any_assessor_role and (not is_qa_complete) and (not flag_status) and not is_expression_of_interest %}
+        {{ assessment_complete(state, form.csrf_token, application_id)}}
+    {% endif %}
+</div>
+</div>
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-one-third">
+        {{ comment(comments) }}
+        {% if display_comment_box != True and not display_comment_edit_box %}
```

Comment:
```suggestion
        {% if not display_comment_box and not display_comment_edit_box %}
```

More pythonic and consistent, but a nitpick so feel free to ignore

Reply:
good shout, resolved.